lane_roi: [[0.16, -0.06, 0.0], [0.3, -0.06, 0.0], [0.3, -0.3, 0.0], [0.16, -0.3, 0.0]]
lane_roi_frame: 'base_link_footprint'
camera_frame: 'camera1_link_optical'
lane_setpoint:  [0.25, -0.2, 0.0]
stop_setpoint:  [0.15, 0.0, 0.0]
window_size:  25
window_count: 8
window_thresh: 5
thresh_val: 100